src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -33,6 +33,7 @@ public FeatureOnOffOptions()
             RenameTracking,
             RefactoringVerification,
             NavigateToDecompiledSources,
+            AlwaysUseDefaultSymbolServers,
```

Comment:
@tmat I tried to follow the pattern with other options work you've done, but wasn't sure about whether the options were still needed to be listed here for some reason, or if I can delete?

Reply:
You can delete these. In fact, you can remove the IOptionProvider and ExportGlobalOptionProvider from this class entirely since these are all global options and as such they don't need IOptionProvider

Reply:
https://github.com/dotnet/roslyn/issues/60082

Reply:
The integration test failure would seem to imply this is not true.. this line is failing:
https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs#L114

Should the tests be doing something different? Or revert this bit for now?

/cc @sharwell

Reply:
nvm, worked out I needed an options provider.